index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+        integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Full stack software Engineer</title>
+</head>
+
+<body>
+    <section class="background-image">
+        <header>
+            <nav class="nav">
+                <div class="container d-flex justify-content-between">
+                    <a href="#" class="my-logo text-decoration-none text--primary weight--700">My Logo</a>
+                    <i class="fa-solid fa-bars text--primary"></i>
+                </div>
+            </nav>
+
+            <div class="container card d-flex align-items-center justify-content-center">
+                <div class="card-title">
+                    <div class="weight--700 nav-heading">I’m Ikechukwu Glad to see you!</div>
+                    <p class="nav-sub-heading weight--400 mt-12">I’m a software developer! I can help you build a
+                        product , feature or website Look through some
+                        of my work and
+                        experience! If you like what you see and have a project you need coded, don’t hestiate to
+                        contact me.</p>
+                    <p class="nav-sub-sub-heading weight--500 text--info mt-12">
+                        LET’S CONNECT
+                    </p>
+
+                    <ul class="mt-12 d-flex list-style-type-none">
+                        <li><i class="fa-brands fa-twitter"></i></li>
+                        <li><i class="fa-brands fa-linkedin-in"></i></li>
+                        <li><i class="fa-brands fa-medium"></i></li>
+                        <li><i class="fa-brands fa-github"></i></li>
+                        <li><i class="fa-brands fa-angellist"></i></li>
+                    </ul>
+                </div>
+            </div>
+            </div>
+        </header>
+
+    </section>
```

Comment:
- [OPTIONAL] It think the `header` tag should only contain the toolbar not the page content and title, in order to respect the semantic rules in the [HTML/CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) :thinking: